### DISCUSSION TITLE
Change srcDir but not work with vue sfc

### DISCUSSION THREAD

**Discussion by wewindy** [ElMassimo/iles#113] [Q&A] [answered]
I init a default project, rename `src` to `gh-pages`, and change the config `srcDir` to `./gh-pages`, but it is not work:

``` typescript
import { defineConfig } from 'iles'

export default defineConfig({
  srcDir: './gh-pages'
})
```

Why?

**Comment by ElMassimo** [comment, marked as answer]
Hi @onsummer, thanks for reporting! [Fixed in `iles@0.7.35`](https://github.com/ElMassimo/iles/commit/d170b4d4d54d0152e723c347f96f465b197a42ad).

This was related to the fact that setting `srcDir` was not affecting the default value of `components.dirs`, which was still set to `src/components`.
